docs/source/using/interacting-with-openshift.adoc

```diff
@@ -24,9 +24,9 @@ For details about managing the Minishift VM, see the link:../using/managing-mini
 The `minishift start` command creates an OpenShift instance using the
 https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md[cluster up] approach.
 
-For this purpose it copies the `oc` binary onto your host. You will find it under `~/.minishift/cache/oc/{openshift-version}/oc`, provided you use Minishift's default version of OpenShift.
+For this purpose it copies the `oc` binary onto your host. It is located in the  _~/.minishift/cache/oc/{openshift-version}_ directory, provided you use Minishift's default version of OpenShift.
```

Comment:
@gbraad I think it is fine as it just point outs the location just for reference and then suggests to use `oc-env` command